aip/general/0111.md

```diff
@@ -0,0 +1,90 @@
+---
+id: 111
+state: draft
+created: 2023-06-10
+placement:
+  category: api-concepts
+  order: 50
+---
+
+# Planes
+
+Resources and methods on an API can be divided into the *plane* that they reside
+or perform operations upon. For the context of APIs, the following planes are
+defined:
+
+- The management plane: a uniform, resource-oriented API that primarily
+  configures and allows retrieval of resources.
+- The data plane: a heteregenous API (ideally resource-oriented) that interface
```

Comment:
This is missing the key aspect of the data plane, which is that it writes and reads user data. In the case of resource-oriented APIs, the data are associated with resources.

Reply:
rephrased to emphasize reading / writing of user data, connected to entities provisioned by the management plane. PTAL.